src/data-workspace/entry-form.js

```diff
@@ -38,6 +44,15 @@ export const EntryForm = React.memo(function EntryForm({ dataSet }) {
 
     return (
         <>
+            {noFormOrLocked?.message && noFormOrLocked?.inForm && (
+                <NoticeBox
+                    className={styles.warningBox}
+                    error={noFormOrLocked?.error}
+                    title={noFormOrLocked?.title}
+                >
+                    {noFormOrLocked?.message}
+                </NoticeBox>
+            )}
```

Comment:
This will never be displayed as the same condition is used in `data-workspace.js` which won't render the `<EntryForm />` when the condition is true.